addons/project/views/project_sharing_views.xml

```diff
@@ -98,8 +98,9 @@
                                 <div class="oe_kanban_bottom_right" t-if="!selection_mode">
                                     <span t-if="record.portal_user_names.raw_value.length > 0" class="pr-2" t-att-title="record.portal_user_names.raw_value">
                                         <t t-set="user_count" t-value="record.portal_user_names.raw_value.split(',').length"/>
-                                        <t t-set="display_nb_assignees" t-value="user_count + ' assignee' + (user_count > 1 ? 's' : '')"/>
-                                        <t t-out="display_nb_assignees"/>
```

Comment:
Even if this would seem highly improbable, in case someone would have x-path any of those, we want to ensure that we do not break the custo. What do you think?
```suggestion
                                        <t t-if="1 == 0" t-set="display_nb_assignees" t-value="user_count + ' assignee' + (user_count > 1 ? 's' : '')"/>
                                        <t t-if="1 == 0" t-out="display_nb_assignees"/>
```